docs/quickstarts/aws.md

```diff
@@ -5,174 +5,226 @@ weight: 10
+>more information. You can determine the version of the Kubernetes control plane included with a
+>Lokomotive release by looking at the [release notes][releases].
+
 
 ## Steps
 
 ### Step 1: Install lokoctl
 
-lokoctl is a command-line interface for Lokomotive.
+`lokoctl` is the command-line interface for managing Lokomotive clusters.
 
-To install `lokoctl`, follow the instructions in the [lokoctl installation](../installer/lokoctl.md)
-guide.
+Download the latest `lokoctl` binary for your platform:
 
-### Step 2: Set up a working directory
+```console
+export os=linux  # For macOS, use `os=darwin`.
 
-It's better to start fresh in a new working directory, as the state of the cluster is stored in this
-directory.
+export release=$(curl -s https://api.github.com/repos/kinvolk/lokomotive/releases | jq -r '.[0].name' | tr -d v)
+curl -LO "https://github.com/kinvolk/lokomotive/releases/download/v${release}/lokoctl_${release}_${os}_amd64.tar.gz"
+```
 
-This also makes the cleanup task easier.
+Extract the binary and copy it to a place under your `$PATH`:
 
 ```console
-mkdir -p lokomotive-infra/myawscluster
-cd lokomotive-infra/myawscluster
+tar zxvf lokoctl_${release}_${os}_amd64.tar.gz
+sudo cp lokoctl_${release}_${os}_amd64/lokoctl /usr/local/bin
+rm -rf lokoctl_${release}_${os}_amd64*
 ```
 
-### Step 3: Set up credentials from environment variables
-
-The AWS credentials file can be found at `~/.aws/credentials` if you have set up and configured AWS
-CLI before. If you want to use that account, you don't need to specify any credentials for lokoctl.
+### Step 2: Create a cluster configuration
 
-You can also take any other credentials mechanism used by the AWS CLI, for example environment
-variables. Either prepend them when starting lokoctl or export each of them once in the current
-terminal session:
+Create a directory for the cluster-related files and navigate to it:
 
 ```console
-AWS_ACCESS_KEY_ID=EXAMPLEID AWS_SECRET_ACCESS_KEY=EXAMPLEKEY lokoctl ...
+mkdir lokomotive-demo && cd lokomotive-demo
 ```
-or
 
-```console
-export AWS_ACCESS_KEY_ID=AKIAIOSFODNN7EXAMPLE
-export AWS_SECRET_ACCESS_KEY=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
-export AWS_DEFAULT_REGION=us-east-1
+Create a file named `cluster.lokocfg` with the following contents:
+
+```hcl
+cluster "aws" {
+  asset_dir            = "./assets"
+  cluster_name         = "lokomotive-demo"
+  controller_count     = 1
+  dns_zone             = "example.com"
+  dns_zone_id          = "ZQC39L1XVW0D"
+  ignore_x509_cn_check = true
+
+  project_id           = "89273817-4f44-4b41-9f0c-cb00bf538542"
+  region               = "us-east-1"
+
+  ssh_pubkeys          = ["ssh-rsa AAAA..."]
+
+  worker_pool "pool-1" {
+    count         = 2
+    instance_type = "i3.large"
+    ssh_pubkeys   = ["ssh-rsa AAAA..."]
+  }
+}
+
+# A demo application.
+component "httpbin" {
+  ingress_host = "httpbin.example.com"
```

Comment:
But I think you will also need contour in this case, if a public URL is needed.

Reply:
DNS is changed as adviced, but I don't think contour is needed. We are port-forwarding for a demo application and we don't need the IP configured.

Reply:
I think we should ask user to install contour as well because it kinda confirms the line in the introduction:

> Lokomotive components complement the "stock" Kubernetes functionality by adding features such as load balancing, persistent storage and monitoring to a cluster. To keep this guide short you will deploy a single component - httpbin - which serves as a demo application to verify the cluster behaves as expected.

Port forwarding is fine for EM or other clouds because they need even more stuff like metallb and external-dns, etc.